nilearn/plotting/html_surface.py

```diff
@@ -164,6 +236,25 @@ def view_img_on_surf(stat_map_img, surf_mesh='fsaverage5',
         If True, image is plotted on a black background. Otherwise on a
         white background. Default=False.
 
+    bg_map : Surface data or 'auto', optional
```

Comment:
Ok I see. Yes then I agree it's fine to leave it like that for now especially since we will start working on a surface API soon. I was just thinking that it could be defined a bit better concerning the current surface object implementation but it's not in the scope of this PR and at least like it's consistent with the other surface functions. Thanks for clarifying!